packages/services/src/setting/index.ts

```diff
@@ -140,6 +151,27 @@ export namespace Setting {
  * A namespace for private data.
  */
 namespace Private {
+  let disabled: { raw: string; rule: RegExp }[] = [];
+  try {
+    let tempDisabled = PageConfig.getOption('disabledExtensions');
+    if (tempDisabled) {
+      disabled = JSON.parse(tempDisabled).map((pattern: string) => {
```

Comment:
Just for my reference: What ended up being the story with pageConfig vs json encoding/not? Will this stay a string, or will this now directly be an object? 😕

Reply:
This will be an object in 2.0.